openjdk/src/test/java/org/conscrypt/SSLUtilsTest.java

```diff
@@ -16,16 +16,18 @@
 
 package org.conscrypt;
 
+import static com.google.common.base.Charsets.UTF_8;
```

Comment:
Just to be clear, `StandardCharsets` is a Java 7 feature. Are you ok with its use here because it's only a test?